tsconfig.api-extractor.json

```diff
@@ -1,6 +1,6 @@
 {
   //TODO: Remove file and reference to it when everything is adjusted to strict mode
-  "extends": "./tsconfig.json",
+  "extends": "./projects/storefrontapp/tsconfig.app.prod.json",
```

Comment:
Thanks to using production `"paths"` mappings in tsconfig, the dependencies types are resolved based on their compiled d.ts files instead of the source code .ts files